app/models/vendor.rb

```diff
@@ -1,3 +1,3 @@
 class Vendor < ApplicationRecord
-
-end
\ No newline at end of file
+has_many :market_vendors, dependent: :destroy
+end
```

Comment:
Will we be using this association in another endpoint as I'm not seeing it used in the code.

Do we need to create/test all associations?

Reply:
The association needs to be set up so that the market_vendor can be destroyed as well when a vendor is destroyed

Reply:
I think any association that we create should be tested, I can add the edge case testing to make it more robust

Reply:
Got it!

Please confirm once you've updated your tests and I can review and merge